cmake/f3dConfig.cmake.in

```diff
@@ -1,9 +1,31 @@
+# When this file is run by CMake through the find_package command, the
+# following targets will exist:
+#
+#   f3d::f3d           Target that contains the F3D executable
```

Comment:
why do we export the F3D executable? Is it making sense? Maybe if a project is creating a custom command calling f3d?

Reply:
yes who knows ? in any case, thisnks automatic, generated by f3dTargets